.github/workflows/publish-package.yml

```
@@ -0,0 +1,36 @@
+name: publish distributions
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - v*
+
+jobs:
+  build-and-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install pep517
+      run: |
+        python -m pip install pep517 --user
+    - name: Build a binary wheel and a source tarball
+      run: |
+        python -m pep517.build --source --binary --out-dir dist/ .
+    - name: Publish distribution 📦 to Test PyPI
+      uses: pypa/gh-action-pypi-publish@v1.0.0a0
+      env:
+        IS_TESTPYPI: ${{ true }}
```

Why use a boolean literal here? Env vars values are always strings so why not just use a string literal?
```suggestion
        IS_TESTPYPI: "true"
```

---

yep, this ended up getting replaced entirely by what I did in kratsg/stare